packages/phishing-controller/src/PhishingDetector.ts

```diff
@@ -201,7 +204,43 @@ export class PhishingDetector {
       }
     }
 
+    // Check for IPFS CID related blocklist entries
+    if (url.match(ipfsCidRegex())) {
+      // there is a cID string somewhere
+      // Determine if any of the entries are ipfs cids
+      // Depending on the gateway, the CID is in the path OR a subdomain, so we do a regex match on it all
+      const cidMatch = url.match(ipfsCidRegex());
+      const cID = cidMatch ? cidMatch[0] : '';
```

Comment:
If you change this to the following it will eliminate one of the coverage problems:

``` suggestion
    const ipfsCidMatch = url.match(ipfsCidRegex());

    // Check for IPFS CID related blocklist entries
    if (ipfsCidMatch !== null) {
      // there is a cID string somewhere
      // Determine if any of the entries are ipfs cids
      // Depending on the gateway, the CID is in the path OR a subdomain, so we do a regex match on it all
      const cID = ipfsCidMatch[0];